tests/Unit/API/Site/Controllers/Ads/CampaignControllerTest.php

```diff
@@ -152,6 +152,67 @@ public function test_get_campaigns_converted_names() {
 		$this->assertEquals( 200, $response->get_status() );
 	}
 
+	public function test_get_campaigns_with_args() {
```

Comment:
In the test added here where are we testing another page_size? For example page_size=1 to see if we don't get all the campaigns and just 1